build.psake.ps1

```diff
@@ -120,6 +120,9 @@ Properties {
     $PublishRootDir = "$PSScriptRoot\.publish"
     $PublishDir     = "$PublishRootDir\$ModuleName"
 
+    # The local installation directory for the install task. Defaults to your user PSModulePath.
+    $InstallPath = Join-Path -Path (Split-Path $profile.CurrentUserAllHosts -Parent) -ChildPath $ModuleName 
+
```

Comment:
So it does, I don't know why I haven't seen that before. You learn something every day!